src/api-service/__app__/onefuzzlib/azure/auto_scale.py

```diff
@@ -123,23 +123,47 @@ def create_auto_scale_profile(min: int, max: int, queue_uri: str) -> AutoscalePr
                 metric_trigger=MetricTrigger(
                     metric_name="ApproximateMessageCount",
                     metric_resource_uri=queue_uri,
-                    # Check every minute
-                    time_grain=timedelta(minutes=1),
+                    # Check every 15 minutes
```

Comment:
Some of these timing numbers were turned after reading this guidance on autoscaling: https://docs.microsoft.com/en-us/azure/architecture/best-practices/auto-scaling

Reply:
Please add this link as comment to the code

Reply:
These are just default values, right? We're (eventually) providing a way for the admin to customize these values? Or do we expect them to use the portal for tweeks.

Reply:
I think the values that we should include in the cli are:
* `max`/`default` - required to create the scale set in the first place
* `min` - optional but defaults to 1
* `scale-out-amount`/`scale-out-cooldown`/`scale-in-amount`/`scale-in-cooldown` - will vary by scale set and use case so it's convenient to have it configurable. Ex: busier systems will want bigger `scale-{in|out}-amount` values, if nodes take long to set up they'll want longer cooldowns.

We can keep the current values as defaults since I think they're appropriate for a less busy deployment.